packages/grid/x-data-grid/src/hooks/features/scroll/useGridScroll.ts

```diff
@@ -30,13 +31,13 @@ function scrollIntoView(dimensions: {
   const elementBottom = offsetTop + offsetHeight;
   // Always scroll to top when cell is higher than viewport to avoid scroll jump
   // See https://github.com/mui/mui-x/issues/4513 and https://github.com/mui/mui-x/issues/4514
-  if (offsetHeight > clientHeight) {
+  if (Math.abs(offsetHeight) > Math.abs(clientHeight)) {
```

Comment:
Why would a client height or an offset height ever be negative? It likely hides something

Reply:
it's not logical for height to be negative unless it's inverted. should I rename the variables?